pkg/asset/cluster/ovirt/ovirt.go

```diff
@@ -10,12 +10,12 @@ import (
 
 // Metadata converts an install configuration to ovirt metadata.
 func Metadata(config *types.InstallConfig) *ovirt.Metadata {
-	customImage, ok := os.LookupEnv("OPENSHIFT_INSTALL_OS_IMAGE_OVERRIDE")
+	_, ok := os.LookupEnv("OPENSHIFT_INSTALL_OS_IMAGE_OVERRIDE")
 	m := ovirt.Metadata{
 		ClusterID: config.Ovirt.ClusterID,
-		// if we have a custom image, don't remove the template, otherwise its
-		// a per deployment template, destroy it
-		RemoveTemplate: ok && customImage != "",
+		// if we have a custom image, don't remove the template,
+		// otherwise its a per deployment template, destroy it
+		RemoveTemplate: !ok,
```

Comment:
Hi Gal, just curious, why are you inverting ok with not ?  IMO, should be just ok. 

Example of RemoveTemplate usage:
https://github.com/openshift/installer/blob/c91e6564f07b03d2c36530858409013b261998d4/pkg/destroy/ovirt/destroyer.go#L125

Reply:
Hi Douglas, I think it's correct what this PR is doing. 
If you have a custom image, you have the `OPENSHIFT_INSTALL_OS_IMAGE_OVERRIDE` defined in your env (the lookup `ok` variable is true) and, according to the code comment, we don't wont to remove the template in case the user has a custom image.
Otherwise the `ok` will be false so we need to set RemoveTemplate to true.

Reply:
Exactly,
OPENSHIFT_INSTALL_OS_IMAGE_OVERRIDE is present --> ok = True --> RemoveTemplate = False --> Template will not be removed and that is what we want because that is a template the installation didn't create
OPENSHIFT_INSTALL_OS_IMAGE_OVERRIDE is not present  --> ok = False --> RemoveTemplate = True --> Template will  be removed and that is what we want because that is a template the installation created

Reply:
Thanks for the clarification guy.  Looks sane to me too :)